Google AI.js

```diff
@@ -0,0 +1,188 @@
+{
+	"translatorID": "5f1c4a3b-b7cf-4170-a896-e4d82c0621c9",
+	"label": "Google AI",
+	"creator": "Guy Aglionby",
+	"target": "^https://ai\\.google/research/(pubs|people)",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2018-09-09 17:55:01"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2018 Guy Aglionby
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero.  If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+var bibtex2zoteroTypeMap = {
+	"inproceedings": "conferencePaper",
+	"conference"   : "conferencePaper",
+	"article"      : "journalArticle",
+	"techreport"   : "report",
+	"patent"       : "patent"
+};
+
+function detectWeb(doc, url) {
+	if (url.includes('/pubs/pub')) {
+		let bibtex = extractBibtex(doc);
+		let doctype = bibtex.split('{')[0].replace('@', '');
+		return bibtex2zoteroTypeMap[doctype];
```

Comment:
I wonder if it's worth putting in a fallback here? Just in case they add an unforeseen item type -- we'd detect the wrong type, but at least we'd show a save icon and the actual import would likely be correct.

Reply:
Could it be worth copying in the whole list from [BibTeX.js](https://github.com/zotero/translators/blob/2d3e4bdbaa217b31eee10591e84d6afb0cd2efb2/BibTeX.js#L200)? Or is there a cleaner way of doing the mapping? I only added types that I saw, but it seems like they put a variety of content on there so adding this and a fallback seems a good idea.

Reply:
Good point. I don't see any real cost of adding the whole list. It's pretty short. I'd still do a fallback, likely to journal article, just in case because bibtex is so poorly standardized. (and sorry for the delay, I could have written those 2 sentences faster ;)

Reply:
Didn't know it was so poorly standardised -- good to know. Amended as suggested, and no problem about the delay